ServerCore/Properties/ServiceDependencies/PuzzleServerTestDeploy - Web Deploy/profile.arm.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Completely unqualified to review this file.

Reply:
This is an autogenerated file for deployment (confirmed there are no secrets in it). I didn't mean to include it in this review, but if we get these checked in, those of us who deploy could avoid having to keep working around them. Is that ok?